### PR TITLE
Add Hindi localisation

### PR DESCRIPTION
**Pull request by anupdsouza**
# What it Does
* #7 
* Add support for Hindi localisation

# How I Tested
* Run the application with device language set to Hindi
* Did basic sanity of all screens

# Notes
* May need further updates as more features are developed and strings are introduced

### CODE REVIEW

File: Basic-Car-Maintenance/Shared/Localizable.xcstrings [hunk 1]
```diff
@@ -75,6 +81,12 @@
             "value" : "🦄 Mikaela Caron - Maintaineur"
           }
         },
+        "hi" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "🦄 मिकाएला कैरोन - अनुरक्षक"
```
Comment (mikaelacaron) [inline Basic-Car-Maintenance/Shared/Localizable.xcstrings]: Is this common to also change my name? and why does Google Translate translate this as "escort?" `अनुरक्षक`

File: Basic-Car-Maintenance/Shared/Localizable.xcstrings [hunk 2]
```diff
@@ -827,7 +1000,14 @@
...
-
+      "localizations" : {
+        "hi" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "लोरेम इप्सम केवल मुद्रण और टाइपसेटिंग उद्योग का अस्थायी पाठ है। लोरेम इप्सम 1500 के दशक से ही उद्योग का मानक अस्थायी पाठ रहा है, जब एक अज्ञात प्रिंटर ने एक प्रकार की गैली ली और उसे एक प्रकार की नमूना पुस्तक बनाने के लिए तैयार किया। यह न केवल पाँच शताब्दियों तक जीवित रहा है, बल्कि इलेक्ट्रॉनिक टाइपसेटिंग में भी छलांग लगाता रहा है, मूलतः अपरिवर्तित रहा है। इसे 1960 के दशक में लोरेम इप्सम अंशों वाले लेट्रासेट शीट के रिलीज के साथ लोकप्रिय बनाया गया था, और हाल ही में लोरेम इप्सम के संस्करणों सहित एल्डस पेजमेकर जैसे डेस्कटॉप प्रकाशन सॉफ्टवेयर के साथ इसे लोकप्रिय बनाया गया था।"
+          }
+        }
+      }
```
Comment (mikaelacaron) [inline Basic-Car-Maintenance/Shared/Localizable.xcstrings]: Don't translate this, it's going to be removed, and is just filler text

**Review by mikaelacaron — CHANGES_REQUESTED** [review]
I have a couple of questions, but otherwise looks good.

File: Basic-Car-Maintenance/Shared/Localizable.xcstrings (hunk 1 above)
Comment (anupdsouza) [reply Basic-Car-Maintenance/Shared/Localizable.xcstrings]: The Hindi script, also called Devanagari is capable of translating names as well. I'm not sure why Google is translating maintainer to escort in English; I think they mean 'protector'. The synonym is actually "maintainer", "care taker". If you'd like, I can keep it in English.

File: Basic-Car-Maintenance/Shared/Localizable.xcstrings (hunk 2 above)
Comment (anupdsouza) [reply Basic-Car-Maintenance/Shared/Localizable.xcstrings]: Sure, I'll revert this once you can confirm the other change as well.

File: Basic-Car-Maintenance/Shared/Localizable.xcstrings (hunk 1 above)
Comment (mikaelacaron) [reply Basic-Car-Maintenance/Shared/Localizable.xcstrings]: @anupdsouza I'll put it up to your best judgement, if you think it the word makes sense in this context, then it's fine to leave it

File: Basic-Car-Maintenance/Shared/Localizable.xcstrings (hunk 2 above)
Comment (mikaelacaron) [reply Basic-Car-Maintenance/Shared/Localizable.xcstrings]: After you've made the updates click the re-review button please

Re-review button

File: Basic-Car-Maintenance/Shared/Localizable.xcstrings (hunk 1 above)
Comment (anupdsouza) [reply Basic-Car-Maintenance/Shared/Localizable.xcstrings]: I've updated it to a more suitable translation

File: Basic-Car-Maintenance/Shared/Localizable.xcstrings (hunk 2 above)
Comment (anupdsouza) [reply Basic-Car-Maintenance/Shared/Localizable.xcstrings]: Done, I've removed the unused translation.

**Review by mikaelacaron — APPROVED** [review]
Great work!

**Pull request merged**